lib/archethic/contracts/interpreter/transaction_statements.ex

```diff
@@ -1,12 +1,17 @@
 defmodule Archethic.Contracts.Interpreter.TransactionStatements do
-  @moduledoc false
+  @moduledoc """
+  Two types of SC functions: Utilities and Statements.Utilities functions include hash,etc.
```

Comment:
I find this comment more troubling than none, There's no Utilities nor Statements.Utilities module

Reply:
Utils in SC, and. full stop of sentence..